src/Arcus.BackgroundJobs/CloudEvent/CloudEventBackgroundJob.cs

```diff
@@ -12,12 +12,12 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 
-namespace Arcus.WebApi.Jobs
+namespace Arcus.BackgroundJobs.CloudEvent
```

Comment:
Do we use this, `Arcus.BackgroundJobs` or `Arcus.BackgroundJobs.Core`?

Reply:
IDK, I used this specific sub-namespace because it would maybe make more sense when other types of jobs are added. Each in a sub-topic-namespace?

Reply:
Sounds good to me!